packages/components/src/components/hds/table/th-selectable.hbs

```diff
@@ -4,13 +4,18 @@
 }}
 
 <Hds::Table::Th class="hds-table__th--is-selectable" @scope={{@selectionScope}} ...attributes>
-  <Hds::Form::Checkbox::Base
-    id={{this.checkboxId}}
-    class="hds-table__checkbox"
-    checked={{@isSelected}}
-    aria-label={{this.ariaLabel}}
-    {{did-insert this.didInsert}}
-    {{will-destroy this.willDestroyNode}}
-    {{on "change" this.onSelectionChange}}
-  />
+  <div class={{this.innerWrapperClassNames}}>
+    <Hds::Form::Checkbox::Base
+      id={{this.checkboxId}}
+      class="hds-table__checkbox"
+      checked={{@isSelected}}
+      aria-label={{this.ariaLabel}}
+      {{did-insert this.didInsert}}
+      {{will-destroy this.willDestroyNode}}
+      {{on "change" this.onSelectionChange}}
+    />
+    {{#if @showSortButton}}
+      <Hds::Table::ThButtonSort @sortOrder={{@sortOrder}} @onClick={{@onClickSort}} />
```

Comment:
Issue: something is missing because the label for aria-labelledby is only saying "sort by ascending". i think its because there is no visible label in the cell.  it should say "sort by selected ascending".

https://github.com/hashicorp/design-system/blob/a5e990eda421353d1f20f9c546c99fa8e5547780/packages/components/src/components/hds/table/th-button-sort.hbs#L9

^^ here you can see the labelledby is the prefix label id + the label id + the suffix id.